.babelrc

```diff
@@ -1,5 +1,12 @@
 {
     "presets": [
-        "es2015"
+        [
+            "@babel/preset-env",
+            {
+                "targets": {
```

Comment:
Does this same "targets" concept need to be added to the application-flavors repository's .babelrc as well?  I wrote that recently and may've missed that this was a requirement.

Reply:
Made .babelrc the same across repos